pkg/config/images.go

```diff
@@ -39,12 +21,10 @@ func openShiftVersion(imageVersion string) (string, error) {
 func (g *simpleGenerator) selectNodeImage(cs *api.OpenShiftManagedCluster) {
 	c := &cs.Config
 	c.ImagePublisher = "redhat"
```

Comment:
ImagePublisher is overriden by the config template so you probably want to set this here in case it's empty (otherwise the template value is never applied correctly).

Reply:
deleted in recent changes.